src/pages/docs/[[...slug]]/index.tsx

```diff
@@ -82,6 +149,20 @@ export async function getStaticProps({ params }) {
           .slice(index + 1)
           .filter(doc => !isCurrent(doc))
           .find(isComponentDoc)?.slugAsParams || null,
+      name: packageJSON.name,
+      title: packageJSON.config?.title,
+      category: packageJSON.config?.category,
+      license: packageJSON.license,
+      description: packageJSON.description,
+      keywords: packageJSON.keywords,
+      version: packageJSON.version,
+      packageUrl: `${packageJSON.repository.url.replace('.git', '')}/tree/main/${
+        packageJSON.repository.directory
+      }`,
+      docUrl: `${projectPackage.repository.url.replace('.git', '')}/edit/main/src/${
+        currentDoc._raw.sourceFilePath
+      }`,
+      bugReportUrl: `${packageJSON.bugs.url}`,
```

Comment:
I think that would be much clean to just pass the packageJSON as prop 🤔 and then you can do the small mappings in the components

Reply:
is has an error when `One of your pages includes a large amount of page data (>= 128kB)`
https://nextjs.org/docs/messages/large-page-data